_posts/2022-05-26-socratic-seminar-112.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+type: socratic
+title: "Socratic Seminar 112"
+#meetup: TODO
+---
+
+## Announcements
+Please join us for our next Socratic Seminar. A special thank you to our sponsors [BNY Mellon](https://www.bnymellon.com/), [CardCoins](https://cardcoins.co), [Chaincode Labs](https://chaincode.com) and [ZEBEDEE](https://zebedee.io) for food, refreshments and event space.
+
+## Presentation
+
+- [Rachel Rybarczyk](https://twitter.com/_rrybarczyk) on Stratum v2
+
+## Mailing Lists  Bitcoin Optech
+
+### Mailing Lists
+
+#### bitcoin-dev
+
+#### lightning-dev
+
+#### dlc-dev
+
+### Optech
+
+### Meetings
+- Bitcoin PR Review Club
+- Bitcoin Core general developer meetings
+- Bitcoin Core wallet meetings
+- dlc-specs meetings
+- Lightning specification meetings
+
+## Network Data
+
+## Research
+
+## InfoSec
+
+## Pull Requests and repo updates
+
+### Bitcoin Core
+
+### eclair
+
+### Core Lightning
+
+### lnd
+
+### rust-lightning
+
+### BOLTS
+
+### New Releases
+
+## Events and Podcasts
+
+## Mining
+
```

Comment:
[Bitcoin ASIC Prices Have Fallen to the Lowest Levels since 2021, research from Luxor's Hashrate Index shows](https://blog.hashrateindex.com/bitcoin-mining-asic-prices-have-revisited-post-china-ban-lows/)
[Bitcoin Hashrate Returns to China, new research from the Cambridge Centre for Alternative Finance shows](https://www.jbs.cam.ac.uk/insight/2022/bitcoin-mining-new-data-reveal-a-surprising-resurgence/)
[Economics of Immersion Cooling for Bitcoin Miners, research from Braiins Mining Insights](https://braiins.com/blog/economics-immersion-cooling-bitcoin-miners)
[Texas Bitcoin Miners Turn Off to Conserve Power for the Grid](https://finance.yahoo.com/news/texas-bitcoin-miners-turn-off-175400091.html)
[The Future of Energy Markets According to Bitcoin Miners - Notes from the Mining Stage at Bitcoin 2022](https://bitcoinmagazine.com/markets/future-of-energy-markets-bitcoin-miners)